src/cephadm/cephadm

```diff
@@ -2039,7 +2040,7 @@ def get_daemon_args(ctx, fsid, daemon_type, daemon_id):
             '--setgroup', 'ceph',
             '--default-log-to-file=false',
             '--default-log-to-stderr=true',
-            '--default-log-stderr-prefix="debug "',
+            '--default-log-stderr-prefix=debug ',
```

Comment:
(nit) maintain the quoting around the hanging space char?

Reply:
nope, those quotes need to go